platforms/utility/browser-set.js

```diff
@@ -0,0 +1,50 @@
+const bsBrowserChrome = {
+  base: 'BrowserStack',
+  browser: 'Chrome',
+  os: 'Windows',
```

Comment:
Just for my own curiosity - Is there any strategy behind running Chrome on Windows and Firefox on OS X?